build.sbt

```diff
@@ -174,6 +174,7 @@ lazy val storage = (project in file("storage"))
       // User can provide any 2.x or 3.x version. We don't use any new fancy APIs. Watch out for
       // versions with known vulnerabilities.
       "org.apache.hadoop" % "hadoop-common" % "3.3.1" % "provided",
+      "org.apache.hadoop" % "hadoop-aws" % "3.3.1" % "provided",
```

Comment:
Can you add a comment explaining why this is marked as provided? Since it is only used for 1 optional feature, we should document it here.